database/factories/LegiScan/BodyFactory.php

```diff
@@ -16,18 +16,18 @@ public function definition()
     {
         static $increment = 1;
 
-        $body_name = ucfirst($this->faker->text(80));
-        $role_name = ucfirst($this->faker->text(80));
+        $bodyName = ucfirst($this->faker->text(80));
```

Comment:
What's important about `80`?... Why can't these just be `'body_name' => $this->faker->sentence(),`?... Not obvious. Is there a good reason or you just picked a random number?...

Reply:
Body name is like: Senate, House of Representatives, House of Delegates, etc. ... I guess just to keep it short? Technically, it can be any string.

Reply:
I will just use the `sentence()` method you have. Working on it.